docs/api/power-monitor.md

```diff
@@ -39,3 +39,23 @@ Emitted when the system changes to AC power.
 ### Event: 'on-battery' _Windows_
 
 Emitted when system changes to battery power.
+
+## Methods
+
+The `powerMonitor` module has the following methods:
+
+#### `shell.querySystemIdleState(idle_threshold, callback)`
+
+* `idleThreshold` Interger
+* `callback` Function
+  * `active` | `idle` | `locked` | `unknown` String
+
+Calculate system idle state. `idle_threshold` is the amount of time (in seconds)
+before considered idle. State will be notified via callback represents current system
+idle state. `locked` is available on supported system only.
```

Comment:
Can we document which systems currently support `locked`?

Reply:
chromium doesn't document it as well (https://cs.chromium.org/chromium/src/ui/base/idle/idle.h?q=idle&sq=package:chromium&l=17) - I'd rather in sync with chromium as it can change anytime due to platform side patch / version up, etcs.